app/src/main/res/layout-sw600dp-land/ongoing_topic_item.xml

```diff
@@ -55,17 +55,16 @@
 
         <TextView
           android:id="@+id/topic_name_text_view"
+          style="@style/TextViewStart"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
           android:layout_marginStart="16dp"
           android:layout_marginTop="8dp"
           android:layout_marginEnd="16dp"
           android:ellipsize="end"
           android:fontFamily="sans-serif"
-          android:gravity="start"
```

Comment:
Both gravity & text alignment is used here--which is correct?

Reply:
android:gravity="start" doesn't work for RTL. This line doesn't seem to make any difference it is unused.